src/database/queries/canonicalFields/fetchAll.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+  canonical_fields.id as "id",
+  canonical_fields.label as "label",
+  canonical_fields.short_code as "shortCode",
+  canonical_fields.field_type as "fieldType",
+  canonical_fields.created_at as "createdAt"
+FROM canonical_fields
```

Comment:
Maybe `from canonical_fields cf` or `from canonical_fields as cf` so we can say `cf.label` instead. Or again, just omit the qualification entirely since there are no other tables nor joins.

Reply:
Totally fine doing either -- which would you prefer?

Reply:
I guess `canonical_fields cf` but it's fine as-is too.